lazy_dataset/core.py

```diff
@@ -1281,6 +1285,9 @@ def batch_dynamic_time_series_bucket(
             expiration: maximum life time of a bucket. After this number of
                 subsequent examples it is either emitted
                 (if drop_incomplete is False) or discarded
+            max_buffer_size: maximum number of examples to be buffered.
```

Comment:
Suggestion:
`max_buffer_size: maximum number of buckets to be buffered.` (Replace examples with buckets)

Reply:
Thanks. Probably a minor point: Adding a hint, that the intent of `expiration` and `max_buffered_examples` are similar and usually only one of them is used?